### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Create new file_Screen Reader: Narrator is reading the "Kind" drop-down as table

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4281]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS1.3.1; #ScreenReader; #Narrator; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev2;

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**Screen Reader:** Narrator

**MAS Violated:** MAS1.3.1

**Repro Steps:**
1.Start Screen Reader. Open Java EE - Eclipse application
2.Install "Azure Toolkit for Eclipse" and "Scala Plug-ins" from Eclipse Marketplace
3.Tab till "File > New > Project" in the navigation bar and press enter
4.Tab till "HDInsight Project > Spark on HDInsight (Scala)" and select it.
5.Tab till "Next" button and press enter 
6.Enter all the required details, tab till "Next" button and press enter
7.Tab till "Module Dependencies" tab and press enter
8.Tab till "Finish" button and press enter
9.In Project Explorer, expand the created Scala project right-click src, New-->Other and press enter
10.Tab till Scala Wizards, expand and select 'Scala Object' and press next
11.Check if the screen reader is reading the "Kind" drop-down properly

**Actual Result:**
Narrator is reading the "Kind" drop-down as table

**Expected Result:**
Narrator should read the "Kind" drop-down as a drop-down

**NOTE:** NVDA is reading the role properly

**User impact:**
Screen Reader users will find it difficult to access the drop-down if the role is read incorrectly as table as the screen reader users might use table shortcuts to access the drop-down

**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/doc2.aspx?sourcedoc=%7B97358E88-4367-4121-BF9D-F0C6597D03B7%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships%20.docx&action=default&mobileredirect=true)

[Bug_Java Tooling_Create a new File_Table_Narrator.pptx](https://github.com/microsoft/azure-tools-for-java/files/4461283/Bug_Java.Tooling_Create.a.new.File_Table_Narrator.pptx)

![Bug_4281_Java Tooling_Create new file_Role of Dropdown_NVDA_No issue](https://user-images.githubusercontent.com/41108985/78987246-1e56f380-7b4b-11ea-8b30-a2cc487728d6.png)

**Comment by t-rufang** [comment]
@soumyatippapuram This issue is duplicate with #4282

**Comment by soumyatippapuram** [comment]
@t-rufang #4281 and #4282 are regarding the same control but both of them are not same issues. 
#4281 is regarding screen reader reading the improper role and #4282 is regarding the screen reader not reading the label

**Comment by jingyanjingyan** [comment]
This wizard is extends from original Scala Plugin. You can find the same dialog in New Project > Scala Wizards > Scala Project. So it's out of the scope of our toolkit.

**Comment by jingyanjingyan** [comment]
+ Importance of experience: Important. Screen Reader users will find it difficult to access the drop-down if the role is read incorrectly as table as the screen reader users might use table shortcuts to access the drop-down
+ Quality of experience: Annoying. It is not block the create process, but might cause user use table shortcuts to access items.
+ Alternate solutions: This issue will not break users experience, no need for alternate solutions
+ Evidence: All evidence is provided in this issue.
+ [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

External issues tracked in: https://bugs.eclipse.org/bugs/show_bug.cgi?id=562990

**Comment by Alexandria-R** [comment]
Approved to close.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749643
